app/components/channel_drawer/channels_list/filtered_list/filtered_list.js

```diff
@@ -21,6 +21,11 @@ import {displayUsername} from 'mattermost-redux/utils/user_utils';
 
 import ChannelDrawerItem from 'app/components/channel_drawer/channels_list/channel_item';
 
+const VIEWABILITY_CONFIG = {
```

Comment:
can we make this in a constant file? I see this used in three different places

Reply:
Will do.